tests/end-to-end/api/04-direct-message.js

```diff
@@ -477,4 +478,141 @@ describe('[Direct Messages]', function() {
 				.end(done);
 		});
 	});
+
+	describe('/im.create', () => {
+		let otherUser;
+		before(async () => {
+			otherUser = await createUser();
+		});
+
+		after(async () => {
+			await deleteUser(otherUser);
+			otherUser = undefined;
+		});
+
+		it('creates a DM between two other parties (including self)', (done) => {
+			request.post(api('im.create'))
```

Comment:
I wonder if this room should be removed afterwards along with `otherUser` that will be removed on the `after` callback of this test.

this is no big deal though, just asking your opinion =)

Reply:
Yes that's a fair call. Just deleting the user will remove only the subscriptions and not the room itself. I will fix it up. 

I note that there is no `deleteRoom` helper defined in tests/data/rooms.helper.js  Would your preference be to add such a method, or just call to the API directly?

Reply:
I can see the benefit of creating a `deleteRoom` helper along with the `createRoom` one =)